Framework/PythonInterface/plugins/algorithms/WorkflowAlgorithms/CorelliPowderCalibrationCreate.py

```diff
@@ -0,0 +1,576 @@
+# Mantid Repository : https://github.com/mantidproject/mantid
+#
+# Copyright &copy; 2018 ISIS Rutherford Appleton Laboratory UKRI,
+#   NScD Oak Ridge National Laboratory, European Spallation Source,
+#   Institut Laue - Langevin & CSNS, Institute of High Energy Physics, CAS
+# SPDX - License - Identifier: GPL - 3.0 +
+import numpy as np
+# import os  # Test to see if the build error goes away
```

Comment:
Perhaps we should remove outdated comments, especially when they have served their purpose during the development cycle.

Reply:
thanks for catching this one. Flew past my radar